package.json

```diff
@@ -199,6 +199,7 @@
     "semver": "^7.7.1",
     "shelljs": "0.8.5",
     "sinon": "7.3.2",
+    "sinon-chai": "3.7.0",
```

Comment:
With this not being in the base package - I'm not sure where the `scripts/unit` tests were pulling in this version. Adding here so it's not confused after the update.